data-serving/data-service/src/controllers/case.ts

```diff
@@ -424,3 +424,35 @@ export const listPlacesOfTransmission = async (
         return;
     }
 };
+
+/**
+ * List most frequently used occupations.
+ *
+ * Handles HTTP GET /api/cases/occupations.
+ */
+export const listOccupations = async (
+    req: Request,
+    res: Response,
+): Promise<void> => {
+    const limit = Number(req.query.limit) || 5;
+    if (limit < 1) {
+        res.status(422).json('limit must be > 0');
+        return;
+    }
+    try {
+        const occupations = await Case.aggregate([
+            { $sortByCount: '$demographics.occupation' },
+            { $sort: { count: -1, _id: 1 } },
```

Comment:
nit, but using the string/name value in lieu of `_id` will make this behavior consistent between different environments of our application that have identical data.

Reply:
The $sortByCount operation returns objects of a form something like `{_id: 'occupationName', count: 5}` so that can't be changed here.